conformance/src/type_checker.py

```diff
@@ -365,7 +365,7 @@ def get_version(self) -> str:
 
     def run_tests(self, test_files: Sequence[str]) -> dict[str, str]:
         proc = run(
-            ["pyrefly", "check", "--output-format", "min-text", "--no-summary"],
+            ["pyrefly", "check", "--output-format", "min-text", "--summary=none"],
```

Comment:
This is a drive-by fix. The latest version of pyrefly does not seem to support `--no-summary` anymore.